pom.xml

```diff
@@ -260,17 +248,6 @@
           <includeTestClasses>false</includeTestClasses>
         </configuration>
       </plugin>
-      <plugin>
-        <groupId>org.sonatype.plugins</groupId>
-        <artifactId>nexus-staging-maven-plugin</artifactId>
```

Comment:
@cstamas Don't I need this to stage the jars for release?  Or should I use some newer mechanism?